Frontend/src/pages/ExploreSystems.js

```diff
@@ -87,6 +87,7 @@ function ExploreSystems({ width }) {
             setSystems(result.data);
         } catch (error) {
             setSystems(null)
+            alert("Error: CANNOt CONNECT TO DATABASE: Make sure Docker is running correctly")
```

Comment:
Is the lowercase t in CANNOt accidental?

Reply:
Thank you, I fixed it